Gemfile

```diff
@@ -58,6 +58,7 @@ group :development do
   gem 'pry',                                :require => false
   gem 'json_pure', '<= 2.0.1',              :require => false if Gem::Version.new(RUBY_VERSION.dup) < Gem::Version.new('2.0.0')
   gem 'beaker-testmode_switcher'
+  gem 'rototiller', '~> 1.0.0',             :require => false
```

Comment:
Just thought about the placement of this and not sure this is the right location. Seems it would be better down in the system_tests section.